src/simplicial-hott/12-cocartesian.rzk.md

```diff
@@ -66,12 +66,12 @@ this is preferred for usage.
   ( f : dhom B b b' u P e e')
   : U
   :=
-    (b'' : B) → (v : hom B b' b'') → (w : hom B b b'') →
-      (sigma : hom2 B b b' b'' u v w) → (e'' : P b'') →
-      (h : dhom B b b'' w P e e'') →
-      is-contr
-        ( Σ ( g : dhom B b' b'' v P e' e'') ,
-          ( dhom2 B b b' b'' u v w sigma P e e' e'' f g h))
+    ( b'' : B) → (v : hom B b' b'') → (w : hom B b b'')
+    → ( sigma : hom2 B b b' b'' u v w) → (e'' : P b'')
+    → ( h : dhom B b b'' w P e e'')
+    → is-contr
+        ( Σ ( g : dhom B b' b'' v P e' e'')
+        , ( dhom2 B b b' b'' u v w sigma P e e' e'' f g h))
```

Comment:
Presumably you would want to align it like this:
```suggestion
      ( b'' : B) → (v : hom B b' b'') → (w : hom B b b'')
    → ( sigma : hom2 B b b' b'' u v w) → (e'' : P b'')
    → ( h : dhom B b b'' w P e e'')
    → is-contr
        ( Σ ( g : dhom B b' b'' v P e' e'')
        , ( dhom2 B b b' b'' u v w sigma P e e' e'' f g h))
```
Although I don't know if that should be a rule. I wasn't the one to initially introduce the prepending binary operators convention.

Reply:
And that is actually a good example of why indentation should be left up to the user and not handled automatically